lib/themes/dosomething/paraneue_dosomething/js/utilities/Analytics.js

```diff
@@ -44,14 +44,19 @@ function subscribeEvents() {
   // Signup button on pitch page
   const isPitchPage = $('.pitch').length > 0;
   if (isPitchPage) {
+    $('#link--campaign-signup-login').on('click', () => {
```

Comment:
To be more specific, this event only fires when a non already authenticated user clicks the signup button on a pitch page